docs/guides/external-node/docker-compose-examples/mainnet-external-node-docker-compose.yml

```diff
@@ -52,3 +45,6 @@ services:
     command:
       - --enable-snapshots-recovery
 
+volumes:
+  postgres:
```

Comment:
Nit: Would `postgres: {}` work here? Subjectively, it's a cleaner way to specify "empty" options, so that this spec looks intentional (and not, e.g. result of some lines missing).

Reply:
Even their official docs are using it this way:
https://docs.docker.com/compose/compose-file/07-volumes/
But I agree it's nicer

Reply:
corrected